zhaquirks/xiaomi/aqara/plug_eu.py

```diff
@@ -346,3 +346,55 @@ class PlugMAEU01Alt3(PlugMAEU01):
     }
 
     replacement = PlugMAEU01.replacement
+
+class PlugAEU001(PlugAEU001)
```

Comment:
You also can't inherit from the same class that you're creating.
Either use `PlugMAEU01` or `CustomDevice`. `CustomDevice` might make more sense here, since it seems like there's quite a big difference between the older plugs and this H2 plug.